.gitignore

```diff
@@ -47,6 +47,7 @@ swupd
 swupd-client*tar.gz
 tap-driver.sh
 test-driver
+.state
```

Comment:
what is this file for ?

Reply:
So this file is needed to keep track of all the files processed by the data_processing_script.py. If you have a file xyz.json and it has already been processed, that means there is an entry in the csv with its contents. So next time, somebody runs the script again, the script skips processing it again.

On the other hand, if we want to reset and start processing all over again, we will can delete this file.

Reply:
I had a question, Would you rather have this .state file in the same directory as the outout files. One risk is that the setup_script now has a restore option pointed out in README.md which will delete the output directory.